test/e2e/cluster/add_remove_cluster.go

```diff
@@ -0,0 +1,41 @@
+package cluster
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/submariner-io/submariner/test/e2e/dataplane"
+	"github.com/submariner-io/submariner/test/e2e/framework"
+)
+
+var _ = Describe("Test to check adding/removing a new cluster to existing setup", func() {
+	f := framework.NewDefaultFramework("add-remove-cluster")
+
+	It("Should be able to add and remove 3rd cluster", func() {
+		By(fmt.Sprintf("Verifying that initially pods from the other clusters can not communicate to service/pod in cluster 1"))
```

Comment:
As a sanity check, at the start, I think we should call `f.FindNodesByGatewayLabel(framework.ClusterC, true)` and verify there aren't any.

Reply:
If there are any gateway nodes before, f.FindNodesByGatewayLabel at L27 fails complaining that there are 2 GW nodes and test fails. So that in itself is a sanity check IMO.

Reply:
yeah that would be more of an indirect check but I think it would be clearer to check here to verify expected test setup prerequisites.

Reply:
Done